### PR TITLE
Correct Spelling Mistakes

### PR DESCRIPTION
**Pull request by reetpriye**
I have corrected all the spelling mistakes that were present, Kindly merge my PR.
A small contribution to the community :)

### CODE REVIEW

File: October_2020/DS&Algo_Series [hunk 1]
```diff
@@ -10,7 +10,7 @@ All the instructors are already placed in FAANG companies and if you're willing
 
 https://www.youtube.com/playlist?list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ
 
-Thanku,
+Thank You,
 Regards,
```
Comment (Kajol-Kumari) [inline October_2020/DS&Algo_Series]: Can you please revert it?

File: October_2020/DWoC [hunk 2]
```diff
@@ -10,7 +10,7 @@ We hope this encourages collaborative experiences and allows students from all b
 
 For more information. Visit - https://dwoc.io/
 
-Thanku,
+Thank You,
```
Comment (Kajol-Kumari) [inline October_2020/DWoC]: Please revert this as well

**Review by Kajol-Kumari — COMMENTED** [review]
Thanks for taking time to contribute to our repo. Please make the minor modifications and it will be good to go :)

File: October_2020/DS&Algo_Series (hunk 1 above)
Comment (reetpriye) [reply October_2020/DS&Algo_Series]: Yeah Sure! I have changed it. Kindly check.

File: October_2020/DWoC (hunk 2 above)
Comment (reetpriye) [reply October_2020/DWoC]: Yes! Done.

**Comment by Kajol-Kumari** [comment]
Thanks for taking time to contribute :)

**Pull request merged**